cli/test/string_empty_to_null_test.dart

```diff
@@ -0,0 +1,15 @@
+import "package:stax/string_empty_to_null.dart";
+import "package:test/test.dart";
+
+void main() {
+  test("Test if empty string returns null", () {
```

Comment:
As soon as we already know this is a test, we do not need to have `"Test"` in the name of the test. Also let's just state description of test case without explicit result.  
```suggestion
  test("empty string", () {
```